plugins/org.jkiss.dbeaver.model.event/src/org/jkiss/dbeaver/model/websocket/event/WSUserEvent.java

```diff
@@ -18,18 +18,24 @@
 
 import org.jkiss.code.NotNull;
 
-public class WSUserDeletedEvent extends WSAbstractEvent {
+import java.util.List;
+
+public class WSUserEvent extends WSAbstractEvent {
```

Comment:
Create separate event for force closed sessions, do not mix it with user delete event